components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/internal/Suggestion.kt

```diff
@@ -163,8 +163,9 @@ private fun AutocompleteButton(
         colorFilter = ColorFilter.tint(colors.autocompleteIcon),
         contentDescription = stringResource(R.string.mozac_browser_awesomebar_edit_suggestion),
         modifier = modifier
-            .width(24.dp)
-            .height(24.dp)
+            .width(48.dp)
+            .height(48.dp)
+            .padding(12.dp)
```

Comment:
The effective tap area is getting tripled from 24x24 to 72x72, right? Just want to make sure this **much** of a size increase is intentional.

Reply:
No, the size is 48dp with a padding of 12, which leaves the icon to the original size of 24.

Reply:
Can we use size() to replace width(), height() here?